src/TesApi.Web/BatchNodeScriptBuilder.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Text;
+using TesApi.Web.Extensions;
+using TesApi.Web.Runner;
+
+namespace TesApi.Web
+{
+    /// <summary>
+    /// Builder of batch command script to be executed on a Batch node. 
+    /// </summary>
+    public class BatchNodeScriptBuilder
+    {
+        /// <summary>
+        /// Name of the environment variable that contains the path to the task directory
+        /// </summary>
+        public const string BatchTaskDirEnvVarName = "AZ_BATCH_TASK_DIR";
+
+        private const string BatchTaskDirEnvVar = $"${BatchTaskDirEnvVarName}";
+
+        /// <summary>
+        /// Name of the TES runner binary file
+        /// </summary>
+        public const string NodeTaskRunnerFilename = "tRunner";
+        /// <summary>
+        /// Name of the TES runner task definition file
+        /// </summary>
+        public const string NodeRunnerTaskDefinitionFilename = "TesTask.json";
```

Comment:
Can we change this name?  Confusing because it's not a Tes.Models.TesTask .  Maybe "NodeTask.json"

Reply:
Yes, I am planning to do file renaming as a follow item, will include changing the binary filename as well as this file.